src/index.js

```diff
@@ -1,10 +1,17 @@
-import React from 'react';
-import T from 'prop-types';
-import { soundManager } from 'soundmanager2';
+import React, {PropTypes as T} from 'react';
```

Comment:
This has to be changed back to what was before. We're no longer getting `PropTypes` from `react` since they will be removed in future versions. We're getting them from the `prop-types` package.